scripts/load_mock_unit_data.sh

```diff
@@ -4,8 +4,8 @@ set -e
 
 REPO_NAME="sds-schema-definitions"
 
-DOWNLOAD_URL="https://github.com/onsdigital/${REPO_NAME}/archive/refs/heads/main.zip"
-DOWNLOAD_NAME="${REPO_NAME}-main"
+DOWNLOAD_URL="https://github.com/onsdigital/${REPO_NAME}/archive/refs/heads/update-prepop-schema-definitions.zip"
```

Comment:
Reminder to revert

Reply:
This must be reverted now, the branch doesn't exist anymore.